src/pkg/packager/deploy.go

```diff
@@ -475,6 +509,7 @@ func (p *Packager) setupState(ctx context.Context) error {
 
 		// Try to create the zarf namespace
 		spinner.Updatef("Creating the Zarf namespace")
+		l.Info("creating the Zarf namespace")
```

Comment:
Non-blocking: Can we easily add the namespace name here?

Reply:
We're creating thee Zarf namespace here - the namespace called Zarf